File: rsa/golang/models/responses/verify_response.go
package modelresponses

type VerifyResponse struct {
	Message    string `json:"message"`
	Signature  string `json:"signature"`
	IsVerified bool   `json:"isVerified"`
}

func ToVerifyResponse(message string, signature string, isVerified bool) VerifyResponse {
	return VerifyResponse{
		Message:    message,
		Signature:  signature,
		IsVerified: isVerified,
	}
}

File: rsa/net9/Models/Responses/VerifyResponse.cs
using System.Text.Json.Serialization;

namespace net9.Models.Responses;

public class VerifyResponse
{
    [JsonPropertyName("message")]
    public required string Message { set; get; }
    [JsonPropertyName("signature")]
    public required string Signature { set; get; }
    [JsonPropertyName("isVerified")]
    public required bool IsVerified { set; get; }
}